_posts/2017-11-15-code-review.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "how to do code review, so it rocks"
+date:   2017-11-15 12:00:00 -0400
+categories: culture code-review
+author: Janice Smith
+---
+
+At Forward Financing we do a lot of code review. It's helped our team commit to delivering quality software and it's turned out pretty great. However, at first it can be hard to know how to dive in,
```

Comment:
You can drop the `However`. Doubling it up with your prepositional is a bit awkward.